examples/mugen/retrieval/evaluation.ipynb

```diff
@@ -24,19 +61,38 @@
     "from torchmultimodal.transforms.video_transform import VideoTransform"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To display videos, we'll also define a utility function."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Util for displaying videos\n",
-    "\n",
     "def save_and_display_video(video_filename, video_tensor):\n",
     "    write_video(video_filename, video_tensor, fps=10, video_codec=\"h264\")\n",
     "    return Video(video_filename, embed=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Instantiate VideoCLIP model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we instantiate the VideoCLIP model and load weights from finetuning on the MUGEN dataset. We can set `text_pretrained=False, video_pretrained=False` as those flags will load weights from pretraining the encoders on different datasets (see [source](https://github.com/facebookresearch/multimodal/blob/main/examples/mugen/retrieval/video_clip.py) for more details)."
```

Comment:
You mean "load pre-trained weights and finetune on the MUGEN dataset"?
The weights are not the result of finetuning the MUGEN dataset, right?

Reply:
the weights **are** the result of finetuning on the mugen dataset. there is no additional training/finetuning in this notebook

Reply:
Do you mean to load the pretrained text encoder and video encoder?
Should we set `text_pretrained=True` and the same for video?

https://github.com/facebookresearch/multimodal/blob/1ca663b4451888521610345349ca18ccc72dc730/examples/mugen/retrieval/video_clip.py#L151-L154

Reply:
>"Here we instantiate the VideoCLIP model and load weights from finetuning on the MUGEN dataset. We can set `text_pretrained=False, video_pretrained=False` as those flags will load weights from pretraining the encoders on different datasets

Do you mean MUGEN has fine tuned the weights and here we are just loading their version of the weights?

Reply:
by default, those flags load the weights from pretraining the text encoder on Wikipedia and pretraining the video encoder on Kinetics400. typically, those flags would be set as True when we want to finetune VideoCLIP on a new dataset, such as the MUGEN dataset. for example, those flags are used in the `train.py` script. 

In the case of this notebook, I want to display the model's predictions on the MUGEN dataset. so I manually load in a different set of weights, which are on AWS and have been finetuned on the MUGEN dataset already by the MUGEN authors. 
The `text_pretrained, video_pretrained` flags are set as `False` here because they are incapable of loading one large weights file (e.g., the mugen-finetuned weights) that includes the text encoder, video encoder, and both projection modules. They can only support loading one weights file for the text encoder and one weights file to the video encoder.

I will edit the explanation in the notebook to add that those flags are used for finetuning, not for evaluation.

Reply:
>so I manually load in a different set of weights, which are on AWS and have been finetuned on the MUGEN dataset already by the MUGEN authors.

Are your manual weights cropped from the original weights file? If the user only has the original (presumably large) file, will they be able to repro what you have here? Or do they need to crop the file themselves?